vulnerabilities/migrations/0023_vcid_migration.py

```diff
@@ -0,0 +1,21 @@
+from django.db import migrations
+from django.db.models import Q
+
+from vulnerabilities.utils import build_vcid
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('vulnerabilities', '0022_alter_vulnerability_vulnerability_id'),
+    ]
+
+    def save_vulnerability_id(apps, schema_editor):
```

Comment:
@TG1999 would using a `bulk_update()` be a better approach?
@tdruez  ^ ?

Reply:
Maybe, I've never used it, so you'll have to give it a try and let me know :)

Reply:
Ok, thanks! let me give it a shot with and without